chain/indexer/Cargo.toml

```
@@ -1,11 +1,12 @@
 [package]
 name = "near-indexer"
-version = "0.8.1"
+version = "0.9.1"
```

There is no breaking change, so I suggest we only bump the patch version.

```suggestion
version = "0.8.2"
```

---

I forgot to change version when we were releasing 0.9.0, so this is correct line